src/test/java/io/strimzi/kafka/metrics/YammerMetricsCollectorTest.java

```diff
@@ -21,6 +24,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
Why do we need this annotation? If we assume tests need an empty metrics registry, I'd prefer explicitly clearing the registry between each test, instead of relying on the test running order.

Reply:
Removed